src/Provider/PointCloudProvider.js

```diff
@@ -174,6 +175,23 @@ export default {
         layer.update = PointCloudProcessing.update;
         layer.postUpdate = PointCloudProcessing.postUpdate;
 
+        // override the default method, since updated objects are metadata in this case
+        layer.getObjectToUpdateForAttachedLayers = (meta) => {
```

Comment:
you should add a test will verify the function work as expected
and others clarify / document the data structure used for cloudPoint (as for example, what is the difference this structure `geometryLayer`)

Reply:
Test `getObjectToUpdateForAttachedLayers` is added, is it testing for `geometryLayer`?
What is data structure used for cloudPoint (as for example, what is the difference this structure `geometryLayer`)

Reply:
`getObjectToUpdateForAttachedLayers` always return the same thing, as it's a contract between MainLoop and Layer.

The description is here https://github.com/iTowns/itowns/pull/789/files#diff-61e012c3e00bdc510be33d9eac684b2aR83, and the comment / ref implementation explains the returned types (it's not using jsdoc since it's not exposed anyway).

Reply:
I can not find an explanation of data structure used for cloudPoind,  in itowns

Reply:
See https://github.com/iTowns/itowns/pull/789#discussion_r200361337